src/js_app_credentials.hpp

```diff
@@ -63,6 +63,12 @@ class CredentialsClass : public ClassDefinition<T, realm::app::AppCredentials> {
         {"serverApiKey",     wrap<server_api_key>},
         {"jwt",              wrap<jwt>},
     };
+    
+    static void get_payload(ContextType, ObjectType, ReturnValue &);
+    
+    PropertyMap<T> const properties = {
+        {"payload", {wrap<get_payload>, nullptr}},
```

Comment:
Should it be added to the API doc and TS defs?

Reply:
Great catch. It's [already in the TS defs](https://github.com/realm/realm-js/blob/master/types/app.d.ts#L135) and I'll add it to the API docs.